testing/testcafe/tests/dataGrid/fixedColumns.ts

```diff
@@ -126,3 +128,79 @@ test('Fixed columns should have same width as not fixed columns with columnAutoW
     '#otherContainer',
   );
 });
+
+// T1148937
+test('Hovering over a row should work correctly when there is a fixed column and a column with a cellTemplate (React)', async (t) => {
+  // arrange
+  const dataGrid = new DataGrid('#container');
+  const firstDataRow = dataGrid.getDataRow(0);
+  const firstFixedDataRow = dataGrid.getFixedDataRow(0);
+  const secondDataRow = dataGrid.getDataRow(1);
+  const secondFixedDataRow = dataGrid.getFixedDataRow(1);
+  const { takeScreenshot, compareResults } = createScreenshotsComparer(t);
+
+  // act
+  await t.hover(firstDataRow.element);
+
+  // assert
+  await t
+    .expect(firstDataRow.isHovered)
+    .ok()
+    .expect(firstFixedDataRow.isHovered)
+    .ok()
+    .expect(await takeScreenshot('T1148937-grid-hover-row-1.png', '#container'))
```

Comment:
Let's use ```dataGrid.element``` instead of a CSS selector

```suggestion
    .expect(await takeScreenshot('T1148937-grid-hover-row-1.png', dataGrid.element))
```